src/types/entity-rankings.ts

```diff
@@ -32,3 +32,9 @@ export interface KPIValue<T = Municipality | Region> {
   belowString?: string;
   aboveString?: string;
 }
+
+export type SectorEmissions = {
```

Comment:
Not sure entity-ranking makes sense for this type since it's used for the details page but also not sure where else to put it 😅 maybe in emissions.ts?